src/main/java/games/strategy/triplea/ui/MouseDetails.java

```diff
@@ -33,7 +33,8 @@ public double getY() {
   }
 
   public boolean isRightButton() {
-    return (m_mouseEvent.getModifiersEx() & InputEvent.BUTTON3_DOWN_MASK) != 0;
+    //FOR WHATEVER REASON THIS NEEDS TO ME META_DOWN INSTEAD OF BUTTON3_DOWN !!!!!!!!!!!
```

Comment:
NB: this comment has a context problem. In the diff it makes sense, you see BUTTON3_DOWN_MASK. Without the diff, a reader is only left asking why this comment is bring up BUTTON3_DOWN_MASK at all.